requirements.txt

```diff
@@ -15,3 +15,4 @@ mss
 distro
 dataclasses
 statsd
```

Comment:
we somehow need to disttinguish this, its probably not available on linux & mac, right?